src/System.Private.CoreLib/src/System/Threading/WaitHandle.cs

```diff
@@ -38,11 +34,7 @@ public abstract class WaitHandle : MarshalByRefObject, IDisposable
 
         internal bool hasThreadAffinity;
 
-        private static IntPtr GetInvalidHandle()
-        {
-            return Win32Native.INVALID_HANDLE_VALUE;
-        }
-        protected static readonly IntPtr InvalidHandle = GetInvalidHandle();
+        protected static IntPtr InvalidHandle => new IntPtr(-1);
```

Comment:
This needs to stay as static readonly field - part of the public surface.

Reply:
Good catch.  Thanks.